src/core/qgsjsonutils.cpp

```diff
@@ -238,7 +238,7 @@ json QgsJsonExporter::exportFeaturesToJsonObject( const QgsFeatureList &features
   const auto constFeatures = features;
   for ( const QgsFeature &feature : constFeatures )
   {
-    data["features"].push_back( exportFeatureToJsonObject( feature ) );
+    data[ "features" ].push_back( exportFeatureToJsonObject( feature ) );
```

Comment:
Since this is a cosmetic one, can you use QStringLiteral on "features" and drop 238 in favor of `qgis::as_const`? ;)

Reply:
no literal here, it's a std::strings, ok for the as_const